proposals/specification/access-grants.bs

```diff
@@ -221,6 +221,18 @@ valid for a [=Data Grant=]:
         to the [=Data Grant=] via `interop:hasDataInstance`.
     * Permissions will be set for the [=Access Grant Subject=] on
         each [=Data Instance=].
+* <b>`interop:InheritInstances`</b>
+  *  When the [=Data Grant=] linked via
+     `interop:inheritsFromGrant` has a scope of `interop:SelectedInstances`, and one
```

Comment:
why do we mention `interop:SelectedInstances` here? The parent Data Grant, linked via `interop:inheritsFromGrant`, could also have 'interop:AllInstances` scope.

Reply:
Good catch - have adjusted in most recent commit 👍